Time.php

```diff
@@ -15,7 +15,7 @@
 	return 1;
 }
 
-define( 'DATAVALUES_TIME_VERSION', '0.8.0 alpha' );
+define( 'DATAVALUES_TIME_VERSION', '0.7.1 alpha' );
```

Comment:
-290148

In which cause this should be a pull request from the head revision of the 0.7.0 tag.
Right now this is pulling on top of a bunch of other changes on master https://github.com/DataValues/Time/compare/0.7.0...master
at least one of which is specified in the Changelog as changing behaviour.

This patch should a) be pulled into master without the version changed
AND
b) pulled onto a branch from 0.7.0 with a change of version to 0.7.1 and then tagged